src/main/io/osd.c

```diff
@@ -33,6 +33,31 @@
 
 FILE_COMPILE_FOR_SPEED
 
+#include "drivers/osd_symbols.h"
+#include "common/maths.h"
+#include "io/displayport_msp_bf_compat.h"
+#include "config/parameter_group.h"
+#include "config/parameter_group_ids.h"
+#include "common/typeconversion.h"
+
+//PG_REGISTER_WITH_RESET_TEMPLATE(osdConfig_t, osdConfig, PG_OSD_CONFIG, 7);
```

Comment:
PG register is lower down in the file. These commented out lines should be removed as they are duplicates and add confusion. It would be good if the PG lines were just after the includes/defines, before the function blocks.